### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by KevinSzr** [NRCHKB/node-red-contrib-unifi-os#34] [General]
Hi,
as a beginner here I am not sure where to put my point (issue or discussion or somewhere else).

I tried the UniFi Node, but it would not connect (still "Initializing...").
(UI 6.0.44 on Synology NAS) IP:Port, Username and Password are correct.

Might there be an issue with my controller or am I doing it wrong?

Best regards,
Kevin

**Comment by crxporter** [comment]
Hello, welcome!

Are you running the controller on the Synology? It _might_ be possible to use this node but it's generally built for Unifi OS rather than the "classic" Unifi controller.

I think it might only work with Unifi OS (UDM, UDM-Pro, and Cloud Key Gen 2). The login endpoints are different, among other things.

**Comment by KevinSzr** [comment]
Hi,
thanks so far. If I find no other solution this might me a reason to purchase a cloud key.

Best regards,
Kevin

**Comment by crxporter** [comment]
I'm not going to try to stop you. It wouldn't take much to adapt the code so it works with other controllers but it's just not the main goal here.

[This node](https://flows.nodered.org/node/node-red-contrib-unifi) should be able to run with your controller. I don't know your skill level but you're welcome to have a look into how difficult it would be to adapt things to other controllers.

That being said - cloud key is a slick device. I had one before I went to UDM Pro.

**Comment by Shaquu** [comment]
@KevinSzr how are we here? What are you up to?

**Comment by fladdie** [comment]
Hi,

I am also a total beginner and have some start difficulties.
I think a complete example would be very helpful. In my case I would like to know if the doorbell has been triggered.

**Comment by Shaquu** [comment]
@fladdie we are working on it.
Just to note, UniFi can break our node with software upgrades so be careful with that (try to wait a week or so, so we can test it first).

**Comment by fladdie** [comment]
@Shaquu that sounds great. In the meantime, I have dealt with the example from [here](https://github.com/NRCHKB/node-red-contrib-unifi-os/discussions/30#discussioncomment-1302513) and now understand at least a few things.

**Reply by Shaquu** [reply]
Happy to hear!
